### PR TITLE
Configure images

### PR DESCRIPTION
**Pull request by numansiddique**

### CODE REVIEW

File: ovn_cluster.sh [hunk 1]
```diff
@@ -8,10 +8,10 @@ set -o errexit
 RUNC_CMD="${RUNC_CMD:-docker}"
 
 BASE_IMAGE="ovn/cinc"
```
Comment (dceara) [inline ovn_cluster.sh:10]: I know it's a bit unrelated but could you also remove BASE_IMAGE now that you're touching the IMAGE variables.  I think we never used it anywhere.

**Review by dceara — APPROVED** [review]
If you feel like removing BASE_IMAGE, that'd be great.  In any case the current changes make sense.

File: ovn_cluster.sh (hunk 1 above)
Comment (numansiddique) [reply ovn_cluster.sh]: Ack.  Done

**Review by flavio-fernandes — COMMENTED** [review]
/lgtm

**Pull request merged**